### DISCUSSION TITLE
Motion Masking and setting it at a global level or a camera level is it addative or replacement?

### DISCUSSION THREAD

**Discussion by MrCaspan** [blakeblackshear/frigate#9488] [Ask A Question] [answered]
All of my cameras have a time stamp on them in the top right hand corner and I want to mask that off to free up the CPU from seeing the seconds tick by as motion but a question I had was, if I set a global motion mask setting so that all cameras have this area masked off and only on a few cameras do I want to mask off a tree or a bush. If I create a new mask ONLY for that camera for said tree, does that camera's settings now compleatly ignore that there is a global config or is it addative like it will take the mask from the global setting AND at the camera level? I am assuming as soon as you set a mask at the camera level that it overrides the global and nothign is brought forward..??

**Comment by NickM-27** [comment, marked as answer]
I believe they are replacements currently

**Reply by MrCaspan** [reply]
Thanks for the quick reply. it seems that is the way its working but I just wanted to make sure. Its hard to force motion when testing this :) 
Based on this I am going to create a suggestion that there is an option at a camera level to import the global settings and then add to it!

**Reply by NickM-27** [reply]
there already is one https://github.com/blakeblackshear/frigate/issues/1694

**Reply by MrCaspan** [reply]
Sorry just saw this now. I linked to that one now in mine. I did seach first I promise!!
